src/Microsoft.ML.Data/DataLoadSave/FakeSchema.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
to follow on the list of people that get picky about using sorting, i have seen both Microsoft.ML than System, and vice-versa. 
we gotta sort this out

Reply:
I suggest we use what VS does when you do Ctrl-K-G (sort usings). It sorts alphabetically.

---
In reply to: [218328459](https://github.com/dotnet/machinelearning/pull/931#discussion_r218328459) [](ancestors = 218328459)